src/analysis/complex/phragmen_lindelof.lean

```diff
@@ -742,12 +744,13 @@ begin
   -- Taking the limit as `ε → 0`, we obtain the required inequality.
   suffices : ∀ᶠ ε : ℝ in 𝓝[<] 0, ∥exp (ε * z) • f z∥ ≤ C,
   { refine le_of_tendsto (tendsto.mono_left _ nhds_within_le_nhds) this,
-    apply ((continuous_of_real.mul continuous_const).cexp.smul continuous_const).norm.tendsto',
-    simp, apply_instance },
+    refine
+      ((continuous_of_real.mul continuous_const).cexp.smul continuous_const).norm.tendsto' _ _ _,
+    simp },
   filter_upwards [self_mem_nhds_within] with ε ε₀, change ε < 0 at ε₀,
   set g : ℂ → E := λ z, exp (ε * z) • f z, change ∥g z∥ ≤ C,
-  replace hd : diff_cont_on_cl ℂ g {z : ℂ | 0 < z.re},
-    from (differentiable_id.const_mul _).cexp.diff_cont_on_cl.smul hd,
+  replace hd : diff_cont_on_cl ℂ g {z : ℂ | 0 < z.re} :=
+    (differentiable_id.const_mul _).cexp.diff_cont_on_cl.smul hd,
```

Comment:
Indented `from` is accepted mathlib style. I like `:=` more, but again we probably shouldn't change what is already fine.